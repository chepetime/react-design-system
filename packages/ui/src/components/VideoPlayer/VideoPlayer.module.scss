@use "./../../styles" as styles;

:root {
}

:global :root .dark {
}

.VideoPlayer {
}
