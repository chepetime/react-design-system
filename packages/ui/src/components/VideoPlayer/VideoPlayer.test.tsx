import React from "react";
import { render } from "@testing-library/react";
import { VideoPlayer } from "./VideoPlayer";

describe("Video", () => {
  test("renders the Video component", () => {
    render(<VideoPlayer />);
  });
});
