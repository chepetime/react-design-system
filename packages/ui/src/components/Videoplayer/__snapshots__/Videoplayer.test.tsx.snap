// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Videoplayer matches snapshot 1`] = `
<div>
  <div
    class="Videoplayer"
  >
    <p>
      Videoplayer component working!
    </p>
    <div />
  </div>
</div>
`;
